packages/solid/src/generators/library/files/tsconfig.json.template

```diff
@@ -0,0 +1,33 @@
+{
+  "extends": "<%= offsetFromRoot %>tsconfig.base.json",
+
+  "compilerOptions": {
+    "moduleResolution": "node",
+    "target": "es2017",
+    /**
+      Tell TypeScript to enforce using `import type` instead of `import` for Types.
+     */
+    "importsNotUsedAsValues": "error",
+    "isolatedModules": true,
+    /**
+      To have warnings/errors of the Svelte compiler at the correct position,
```

Comment:
Maybe remove the hint for svelte

Reply:
Ah yeah, copy/paste from svelte config. I will use the same tsconfig as for application